cmd/notation/sign.go

```diff
@@ -22,6 +23,8 @@ const (
 	signatureManifestImage    = "image"
 )
 
+const referrersTagSchemaDeleteError = "failed to delete dangling referrers index"
```

Comment:
@shizhMSFT @Wwwsylvia Since this error is generated based on oras-go-specific procedure, should oras-go export this error so caller can implement upon?

Reply:
It's beyond this PR's scope, I created an [issue ](https://github.com/oras-project/oras-go/issues/479) for tracking, let's move discussion there.

Reply:
Yeah we need to think about how to improve `oras-go` for such use case.

Reply:
Once `oras-go` is updated and have a new release, we can update here. But for now, let's do a string match.